R/download_zenodo.R

```diff
@@ -63,6 +67,10 @@ download_zenodo <- function(doi,
   req <- curl_fetch_memory(paste0(base_url, record))
   content <- fromJSON(rawToChar(req$content))
 
+  # Calculate total file size
+  totalsize <- sum(content$files$size) %>%
+                human_filesize
```

Comment:
```suggestion
                human_filesize()
```
This makes it more clear that is is a function

Reply:
True.